### PR TITLE
Fix Active-Active Gateway feature commandlets

### PR DESCRIPTION
**Pull request by Nilambari**
Changed the bool param: to SwitchParameter to enable/disable Active-Active feature while creating / updating virtual network gateway.
Testing is done and working fine.

### CODE REVIEW

**Comment by Nilambari** [comment]
On demand:- http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1199/

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs [hunk 1]
```diff
@@ -85,9 +85,8 @@ public class NewAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
 
         [Parameter(
             Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
-            HelpMessage = "Active-Active feature flag")]
-        public bool ActiveActive { get; set; }
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs]: @Nilambari this is a breaking change; scripts using the `ActiveActive` parameter for this cmdlet will no longer work.

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs [hunk 2]
```diff
@@ -80,11 +80,21 @@ public class SetAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
             HelpMessage = "The weight added to routes learned over BGP from this virtual network gateway")]
         public int PeerWeight { get; set; }
 
+        //[Parameter(
+        //    Mandatory = false,
+        //    ValueFromPipelineByPropertyName = true,
+        //    HelpMessage = "ActiveActive feature flag")]
+        //public bool ActiveActive { get; set; }
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs]: @Nilambari same comment as above

Comment (Nilambari) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs]: Yes but these command lets are not yet released as in there are no customers who use these command lets yet. We will release the at Ignite. They were just part of last PS release but no customers using it yet.

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs (hunk 1 above)
Comment (Nilambari) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs]: Yes but these command lets are not yet released as in there are no customers who use these command lets yet. We will release the at Ignite. They were just part of last PS release but no customers using it yet.

**Comment by cormacpayne** [comment]
@Nilambari Hey Nilambari, would you mind pulling from dev to get the latest changes that fix the on-demand build?

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs (hunk 1 above)
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs]: It doesn't matter how many customers are using the cmdlet - you need to obsolete parameters, preferrably several months before deleteing them

File: src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs (hunk 2 above)
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs]: Same comment

**Comment by Nilambari** [comment]
Done latest pull from dev branch. On-demand:-  http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1215/

**Comment by cormacpayne** [comment]
@Nilambari approving changes since we are taking a breaking change for this release

**Comment by Nilambari** [comment]
Thanks. Can you Please merge the PR? Thanks!

**Pull request merged**